src/train.jl

```diff
@@ -325,7 +325,7 @@ function sciml_train(loss, opt::BBO = BBO(), data = DEFAULT_DATA;lower_bounds, u
     first(x)
   end
 
-  bboptre = BlackBoxOptim.bboptimize(_loss;Method = opt.method, SearchRange = [(lower_bounds[i], upper_bounds[i]) for i in 1:length(lower_bounds)], MaxSteps = maxiters, kwargs...)
+  bboptre = BlackBoxOptim.bboptimize(_loss;Method = opt.method, SearchRange = [(lower_bounds[i], upper_bounds[i]) for i in 1:length(lower_bounds)], MaxSteps = maxiters, CallbackFunction = cb, CallbackInterval = 0.0, kwargs...)

```

Comment:
what's the callback interval?

Reply:
It controls how often the callback is called, 0.0 implies on every step. Since other optimizers call it on every step I think it should be same here?

Reply:
Makes sense.